janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -156,7 +156,7 @@ public static Mode parse(String mode) {
             ConfigOption.Type.GLOBAL_OFFLINE, "cloud");
 
     public static final ConfigOption<Boolean> DYNAMIC_FIELDS = new ConfigOption<>(SOLR_NS,"dyn-fields",
-            "Whether to use dynamic fields (which appends the data type to the field name). If dynamic fields is disabled" +
+            "Whether to use dynamic fields (which appends the data type to the field name). If dynamic fields is disabled, " +
```

Comment:
This is the source of the output bug, and it's arguably a clear bug as someone is pasting two whitespace-free strings together and expecting a good outcome.